saveJsonFile.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import argparse
+import json
+import pprint
+import requests
+import sys
+import urllib
+
+
+try:
+    # For Python 3.0 and later
+    from urllib.error import HTTPError
+    from urllib.parse import quote
+    from urllib.parse import urlencode
+except ImportError:
+    # Fall back to Python 2's urllib2 and urllib
+    from urllib2 import HTTPError
+    from urllib import quote
+    from urllib import urlencode
+
+
+API_KEY= 'V4mhEirjIuAFM2xPFw9Zx3h-OccaRn079SLXXLPAtjk4yLLUB1v-hKRXLmqfPClCY0uQhIJSl6qhCKpAE7j9LVc14F15ufG4KmMMq-Xo0YidXVQqqiSHmXmSgwhQWnYx'
+
+
+# API constants, you shouldn't have to change these.
+API_HOST = 'https://api.yelp.com'
+SEARCH_PATH = '/v3/businesses/search'
+BUSINESS_PATH = '/v3/businesses/'  # Business ID will come after slash.
+
+
+# Defaults for our simple example.
+DEFAULT_TERM = 'cafe'
+DEFAULT_LOCATION = 'San Francisco, CA'
+SEARCH_LIMIT = 3
+
+
+def request(host, path, api_key, url_params=None):
+    """Given your API_KEY, send a GET request to the API.
+    Args:
+        host (str): The domain host of the API.
+        path (str): The path of the API after the domain.
+        API_KEY (str): Your API Key.
+        url_params (dict): An optional set of query parameters in the request.
+    Returns:
+        dict: The JSON response from the request.
+    Raises:
+        HTTPError: An error occurs from the HTTP request.
+    """
+    url_params = url_params or {}
+    url = '{0}{1}'.format(host, quote(path.encode('utf8')))
+    headers = {
+        'Authorization': 'Bearer %s' % api_key,
+    }
+
+    #print(u'Querying {0} ...'.format(url))
```

Comment:
Please remove

Reply:
Yes, I have many mistake this problem...